libraries/apollo-gradle-plugin-external/src/main/kotlin/com/apollographql/apollo3/gradle/api/Service.kt

```diff
@@ -426,6 +426,22 @@ interface Service {
    */
   val generateOptionalOperationVariables: Property<Boolean>
 
+  /**
+   * Specifies which methods will be auto generated on operations, models, fragments and input objects.
+   *
+   * Pass a list of any of the following:
+   *
+   * - "equalsHashCode" generates `equals` and `hashCode` methods that will compare generated class properties
+   * - "toString" generates a method that will print a pretty string representing the data in the class
+   * - "copy" (Kotlin only) generates a method that will copy the class with named parameters for
+   * - "dataClass" (Kotlin only and redundant with all other methods) generates the class as a [data class](https://kotlinlang.org/docs/data-classes.html)
```

Comment:
`dataClass` felt a little clearer than `data`, which sounded like a method called `data()` to me. I'm happy to go back to the naming in #5207 if you feel strongly.